.markdownlint-cli2.yaml

```diff
@@ -3,3 +3,5 @@ config:
   line-length: false # MD013
   no-duplicate-heading: false # MD024
   reference-links-images: false # MD052
+  first-line-heading: false # MD041
```

Comment:
is this managed by standard files?

Reply:
Yes the docs want circling back to and updating to match without these. But there are a **lot** of doc pages. So a bodge to get this out the door.